api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/MessagePayloadTestDataFactory.java

```diff
@@ -45,8 +45,8 @@ public static RequestNewCardMessagePayload aNewCardRequestMessagePayload(UUID cl
                 .build();
     }
 
-    public static SaveNewCardMessagePayload aSaveCardMessagePayload(UUID claimId) {
-        return SaveNewCardMessagePayload.builder()
+    public static CompleteNewCardMessagePayload aSaveCardMessagePayload(UUID claimId) {
```

Comment:
You need to change the method name here too, should be `aCompleteNewCardMessagePayload`